Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1572,6 +1625,101 @@ def get_impacted_resources(client: PrismaCloudComputeClient, args: dict) -> Comm
         raw_response=raw_response
     )
 
+def get_waas_policies(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Get the WAAS policies.
+    Implement the command 'prisma-cloud-compute-get-waas-policies'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-waas-policies command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    policies = client.get_waas_policies()
+
+    rules = policies["rules"][0]["applicationsSpec"][0]
+    formatted_waas_policy = [{
+        "SQLInjection": rules["sqli"]["effect"],
+        "CrossSiteScriptingXSS": rules["xss"]["effect"],
+        "OSCommandInjetion": rules["cmdi"]["effect"],
+        "CodeInjection": rules["codeInjection"]["effect"],
+        "LocalFileInclusion": rules["lfi"]["effect"],
+        "AttackToolsAndVulnScanners": rules["attackTools"]["effect"],
+        "Shellshock": rules["shellshock"]["effect"],
+        "MalformedHTTPRequest": rules["malformedReq"]["effect"],
+        "ATP": rules["networkControls"]["advancedProtectionEffect"],
+        "DetectInformationLeakage": rules["intelGathering"]["infoLeakageEffect"]
+    }]
+    data = {
+        "Name": "dvwa",
+        "WaasPolicy": formatted_waas_policy
+    }
+
+    return CommandResults(
+        outputs_prefix="PrismaCloudCompute.Policies",
+        outputs_key_field="Name",
+        outputs=data,
+        readable_output=tableToMarkdown(data["Name"], data["WaasPolicy"]),
+        raw_response=policies
+    )
+
+
+def update_waas_policies(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Update the WAAS policy.
+    Implement the command 'prisma-cloud-compute-update-waas-policies'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-update-waas-policies command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    waas_settings = ["sqli", "xss", "attackTools", "shellshock", "malformedReq", "cmdi", "lfi", "codeInjection"]
+
+    policy = args.get("policy")
+
+    for index, rule in enumerate(policy["rules"]):
+        if rule["name"] != args.get("rule_name"):
+            continue
+        policy["rules"][index]["applicationsSpec"][0][args.get("attack_type")] = {"effect": args.get("action") }
+
+    res = client.update_waas_policies(policy)
+    if res.status_code == 200:
+        demisto.results("Successfully updated the WaaS policy")
```

Comment:
please assign to human_readable and return a CommandResults as specified in the doc string, And in compliance with the other functions.